src/CSModule.sol

```diff
@@ -85,6 +86,10 @@ contract CSModule is
     uint64 private _depositableValidatorsCount;
     uint64 private _nodeOperatorsCount;
 
+    ICSStrikes public strikes;
```

Comment:
Maybe we can re-use the old deprecated slot with slashings?

Reply:
In general I think we should find a way to set this address in one contract and then fetch by the others. Not sure how feasible it is, but let's try

Reply:
The only way I see is to change entry-point from `CSModule` to `CSStrikes`. It will help us to get rid of setting address in `CSModule` and just have one more role for calling `ejectBadPerformer`

Reply:
Will implement it